_posts/2015-09-10-Belgian fries.md

```diff
@@ -0,0 +1,14 @@
+#~~French~~ Belgian fries
+
```

Comment:
Hey @faustusdotbe, pretty good start, a complete new recipe. I would just suggest you look at another recipe files and look at the metadata found in the header which is used to catalog each recipe.

Can you add that in?

Apart from that it's perfect.